sphinx/screens-detail/support-ticket/support-ticket/src/main/java/chat/sphinx/support_ticket/ui/SupportTicketViewModel.kt

```diff
@@ -1,15 +1,123 @@
 package chat.sphinx.support_ticket.ui
 
+import android.content.Context
+import android.content.Intent
+import android.net.Uri
+import android.text.Editable
+import androidx.lifecycle.viewModelScope
+import chat.sphinx.concept_network_query_lightning.NetworkQueryLightning
+import chat.sphinx.kotlin_response.LoadResponse
+import chat.sphinx.kotlin_response.Response
 import chat.sphinx.support_ticket.navigation.SupportTicketNavigator
 import dagger.hilt.android.lifecycle.HiltViewModel
-import io.matthewnelson.android_feature_viewmodel.BaseViewModel
+import io.matthewnelson.android_feature_viewmodel.SideEffectViewModel
+import io.matthewnelson.android_feature_viewmodel.submitSideEffect
 import io.matthewnelson.concept_coroutines.CoroutineDispatchers
+import io.matthewnelson.concept_views.viewstate.ViewStateContainer
+import io.matthewnelson.concept_views.viewstate.value
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.collect
+import kotlinx.coroutines.launch
+import javax.annotation.meta.Exhaustive
 import javax.inject.Inject
 
 @HiltViewModel
 internal class SupportTicketViewModel @Inject constructor(
     dispatchers: CoroutineDispatchers,
     val navigator: SupportTicketNavigator,
-): BaseViewModel<SupportTicketViewState>(dispatchers, SupportTicketViewState.Idle)
+    private val networkQueryLightning: NetworkQueryLightning,
+): SideEffectViewModel<
+        Context,
+        SupportTicketSideEffect,
+        SupportTicketViewState>(dispatchers, SupportTicketViewState.Empty)
 {
+    private inner class LogsStateContainer: ViewStateContainer<SupportTicketViewState>(SupportTicketViewState.Empty)
+
+    private val logsStateContainer: ViewStateContainer<SupportTicketViewState> by lazy {
+        LogsStateContainer()
+    }
```

Comment:
`ViewStateConainer<SupportTicketViewState>` is being lazily initialized via the `BaseViewModel` with the initial value of what was passed in it's constructor (`SupportTicketViewState.EMPTY`). it is available from via `viewModel.viewStateContainer`